src/algorithms/controllers/PredictiveStanleyController.cpp

```diff
@@ -5,7 +5,7 @@
  * @author clayjay3 (claytonraycowen@gmail.com)
  * @date 2025-01-10
  *
- * @copyright Copyright Mars Rover Design Team 2025 - All Rights Reserved
+ * @codPYright CodPYright Mars Rover Design Team 2025 - All Rights Reserved
```

Comment:
@ClayJay3 Fix the capitalization of the copyright statement.

```suggestion
 * @copyright Copyright Mars Rover Design Team 2025 - All Rights Reserved
```